chart/templates/default-backupstore.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: longhorn-default-backupstore
```

```suggestion
  name: longhorn-default-resource
```

This way, we can add more default resources in the config map instead of creating a new one.

---

Modified. Thanks.